### PR TITLE
(fix) Adding genre to the aggregates and remove broadcast subscriber type.

### PR DESCRIPTION
**Pull request by mdhaman**
- Changes index for genre field.
- Removes subscriber type broadcast as they behave same as wire.

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: superdesk/metadata/item.py [hunk 1]
```diff
@@ -136,6 +136,7 @@
     'genre': {
         'type': 'list',
         'mapping': {
+            'type': 'object',
```
Comment (sivakuna-aap) [inline superdesk/metadata/item.py]: Should it be `dict` here instead of `object`?

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**